contracts/Core/StakeManager.sol

```diff
@@ -120,9 +120,9 @@ contract StakeManager is Utils, ACL, StakeStorage {
         uint256 stakerId = stakerIds[msg.sender];
         Structs.Staker storage staker = stakers[stakerId];
         require(staker.id != 0, "staker doesnt exist");
-        require(staker.epochLastRevealed == epoch.sub(1), "Didnt reveal in last epoch");
         require(staker.unstakeAfter == 0, "Did not unstake");
         require((staker.withdrawAfter <= epoch) && staker.withdrawAfter != 0, "Withdraw epoch not reached");
+        require((staker.withdrawAfter - Constants.withdrawLockPeriod()) >= staker.epochLastRevealed, "Participated in Withdraw lock period, Cant withdraw");
         require(voteManager.getCommitment(epoch, stakerId) == 0x0, "already commited this epoch. Cant withdraw");
         _givePenalties(staker.id, epoch);
```

Comment:
Hey,
According to this PR, we are not allowing a staker to report any values during withdraw lock to avoid hit and run cases but we are still giving out inactivity penalties over here.

Reply:
Yep this need to be addressed, on it

Reply:
Addressed